src/components/DirectPaymentCreateModal/DirectPaymentCreateModal.tsx

```diff
@@ -288,140 +324,192 @@ export const DirectPaymentCreateModal: React.FC<DirectPaymentCreateModalProps> =
+                    id="walletId"
+                    label="Wallet"
+                    value={formData.walletId}
+                    onChange={handleInputChange}
+                    error={formErrors.walletId}
+                    disabled={isWalletFieldDisabled}
+                    note={getWalletFieldNote()}
+                    required
+                  >
+                    <option value="">Select wallet</option>
+                    {renderWalletOptions()}
+                  </Select>
+                </div>
               </div>
-            </div>
 
-            {/* External Payment ID */}
-            <div
-              className="DirectPaymentCreateModal__fieldWrapper"
-              data-visible={showReceiverField}
-            >
-              <div className="DirectPaymentCreateModal__fieldWrapper__content">
-                <Input
-                  fieldSize="sm"
-                  id="externalPaymentId"
-                  name="externalPaymentId"
-                  type="text"
-                  label="External Payment ID (optional)"
-                  placeholder="Enter external payment ID"
-                  value={formData.externalPaymentId}
-                  onChange={handleInputChange}
-                />
+              {/* External Payment ID */}
+              <div
+                className="DirectPaymentCreateModal__fieldWrapper"
+                data-visible={showReceiverField}
+              >
+                <div className="DirectPaymentCreateModal__fieldWrapper__content">
+                  <Input
+                    fieldSize="sm"
+                    id="externalPaymentId"
+                    name="externalPaymentId"
+                    type="text"
+                    infoText="An identifier for the payment defined by your organization"
+                    tooltipPlacement="top-start"
+                    label="Enter external payment ID (optional)"
+                    placeholder="Enter external payment ID"
+                    value={formData.externalPaymentId}
+                    onChange={handleInputChange}
+                  />
+                </div>
               </div>
             </div>
-          </div>
-        </Modal.Body>
-        <Modal.Footer>
-          <Button size="md" variant="tertiary" type="reset" disabled={isLoading}>
-            Cancel
-          </Button>
-          <Button
-            size="md"
-            variant="primary"
-            type="submit"
-            disabled={isLoading}
-            isLoading={isLoading}
-          >
-            Confirm payment
-          </Button>
-        </Modal.Footer>
-      </form>
+          </Modal.Body>
+          <Modal.Footer>
+            <Button size="md" variant="tertiary" type="reset" disabled={isLoading}>
+              Cancel
+            </Button>
+            <Button
+              size="md"
+              variant="primary"
+              type="submit"
+              disabled={isLoading}
```

Comment:
We don't need to set `disabled` with `isLoading` state because the `isLoading` prop disables the button internally in the SDS.

<img width="742" height="498" alt="image" src="https://github.com/user-attachments/assets/3e15486c-c768-4ebb-8294-1c07a188b29b" />

We should enable the Submit only when all required information is filled out to stay consistent with other forms. For example, the Create New Key modal.

<img width="742" height="1026" alt="image" src="https://github.com/user-attachments/assets/7977d99e-a9da-4a87-b520-65da7d9b1ee4" />